kobo/apps/openrosa/apps/main/migrations/0019_remove_PERM_FROM_KC_ONLY.py

```diff
@@ -0,0 +1,25 @@
+from django.db import migrations
+
+from kpi.models.object_permission import ObjectPermission
+
+PERM_FROM_KC_ONLY = 'from_kc_only'
+
+
+def noop(apps, schema_editor):
+    # Irreversible operation
+    pass
+
+
+def remove_kc_only_perm(apps, schema_editor):
+    deleted = ObjectPermission.objects.filter(
+        permission__codename=PERM_FROM_KC_ONLY
+    ).delete()
```

Comment:
How fast is it on a (really) big production server like ours? If it takes too long, we may need to use the long-running migration mechanism.